.github/workflows/ci.yml

```diff
@@ -55,10 +55,6 @@ jobs:
     - name: Install library
       run: pip install .
 
-    - name: Add folder to PYTHONPATH
-      run: |
-        echo "PYTHONPATH=$PYTHONPATH:$PWD/.." >> "$GITHUB_ENV"
-
```

Comment:
This should no longer be necessary to use the `ml_croissant` package. As far as I can tell, this step served no other purpose than making sure the package was importable.